checks.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+from argparse import ArgumentParser
+from typing import List
+from subprocess import check_output, CalledProcessError
+import sys
+
+IGNORE_PATHS: List[str] = [
+    "jobsapp/migrations",
```

Comment:
Could we change this to catch all possible `migrations` folder? Maybe something like `./*/migrations`
Because there is also the folder `accounts/migrations` and maybe we could have more on the future. Having a regex path we ensure that all of them will be ignored.

Reply:
Shouldn't be that hard to parse regex, 👍